doc/dev/Mgmt-Naming-Conventions.md

```diff
@@ -0,0 +1,99 @@
+# Azure SDK for .NET - Management Plane SDK Naming Conventions
+
+This document describes the naming conventions for the Azure Management Plane SDK for .NET.
+In addition to adhering to the [General Naming Conventions of .NET](https://learn.microsoft.com/dotnet/standard/design-guidelines/general-naming-conventions), there are specific guidelines related to the naming of classes, properties, and methods. These include rules for using abbreviations and acronyms, as well as recommendations for avoiding Azure-specific names.
+
+## General Guidelines
+
+**DO** Use Pascal case which capital case for the first letter only.
+
+**DO** 2 letter acronyms are special which uppercase all if not part of longer compound identifiers, but still have some except like `Vm`, `Id` and etc.  
+
+**DO** Use a singular type name for an enumeration unless its values are bit fields. Use a plural type name for an enumeration with bit fields as values, also called flags enum. For more details, please check [here](https://docs.microsoft.com/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations).
+
+**DO** The name of PATCH operation's body parameter should be: `[Model]Patch`.  For the model of a property in the `[Model]Patch`, if you need to rename the suffix, use `Content` instead of `Patch` since properties are not patchable, they will be set as the property model is.
+
+**DO** The name of PUT / POST operation's body parameter should be: `[Model]Content` or  `[Model]Data`.
+
+**DO** Start property or parameter names of type bool with a verb, like `Is`, `Can`, `Has` prefix.
+
+**DO** End property or parameter names of type DateTime with `On`.
+
+  - Use the original verb form for active voice, like `StartOn` and `EndOn`.
+
+  - Use the passive verb form for passive voice, like `CreatedOn`, `EstablishedOn`. 
+
+**DO** End property or parameter names of type integer that represent intervals or durations with units, for example: `MonitoringInterval` -> `MonitoringIntervalInSeconds`.
+
+**DO** Rename the property or parameter if its name ends with `Interval` or `Duration` and the value is not in ISO 8601 duration format (like P1DT2H59M59S) or ISO 8601 format (like 2.2:59:59.5000000).
```

Comment:
I am confused on this one, rename the property or parameter to what?

Reply:
Any other names that without `Interval` or `Duration` suffix.

Reply:
updated